### DISCUSSION TITLE
How to rewrite the sql statement using prisma syntax below?

### DISCUSSION THREAD

**Discussion by zwl1619** [prisma/prisma#22787] [Q&A]
How to rewrite the sql statement using prisma syntax below?

```
    const ITEMS_PER_PAGE = 6;

    const invoices = await sql`
      SELECT
        invoices.id,
        invoices.amount,
        invoices.date,
        invoices.status,
        customers.name,
        customers.email,
        customers.image_url
      FROM invoices
      JOIN customers ON invoices.customer_id = customers.id
      WHERE
        customers.name ILIKE ${`%${query}%`} OR
        customers.email ILIKE ${`%${query}%`} OR
        invoices.amount::text ILIKE ${`%${query}%`} OR
        invoices.date::text ILIKE ${`%${query}%`} OR
        invoices.status ILIKE ${`%${query}%`}
      ORDER BY invoices.date DESC
      LIMIT ${ITEMS_PER_PAGE} OFFSET ${offset}
    `;
```
I didn't find the corresponding syntax of `amount::text` and `date::text`.

**Comment by ludralph** [comment]
Hi @zwl1619 👋 

Thank you for raising this question.

Do you want to write the SQL statement using `Raw Queries` in Prisma or do you want to try writing the SQL using Prisma Client API? Also can you please provide the schema file for this SQL?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
